scipy/stats/_stats_py.py

```diff
@@ -9649,8 +9669,9 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
 
     Notes
     -----
-    The first Wasserstein distance between the distributions :math:`u` and
-    :math:`v` is:
+    Given two samples (expressed as probability mass functions), :math:`u`
+    and :math:`v`, the first Wasserstein distance between the distributions
```

Comment:
To simplify
```suggestion
    Given two probability mass functions, :math:`u`
    and :math:`v`, the first Wasserstein distance between the distributions
```
It's not immediately clear what it means to express a sample as a probability mass function. If you want to also define the Wasserstein distance means for two samples, I would do that after defining what it means for distributions.